### PR TITLE
EZP-30662: Added http cache tests

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://jira.ez.no/browse/EZP-30662

Related PRs:
https://github.com/ezsystems/ezplatform/pull/436
https://github.com/ezsystems/ezplatform-http-cache/pull/95

This PR:
1) Adds support for Content Editing using Behat steps (for all fieldtypes currently supported by this bundle)
2) Adds a step for browser response header checking
3) Adds a FrontendContext to handle frontend actions (login under siteaccess)

### CODE REVIEW

File: features/examples/content.feature [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+       | Float CT2             | FloatCT2              | Float                        | 2.34                                                                        |
+       | Integer CT2           | Integer2              | Integer                      | 10                                                                          |
+       | Map location CT2      | MapLocationCT2        | Map location                 | Tokio                                                                       |
+       | Date and time CT2     | DateAndTimeCT2        | Date and time                | 2018-12-31 13:55:00                                                         |
+       | Content relation CT2  | ContentRelationCT2    | Content relation (single)    | /RelationFolder1                                                            |
+       | Content relations CT2 | ContentRelationsCT2   | Content relations (multiple) | RelationFolder1,/RelationFolder2                                            |
+       | Image CT2             | ImageCT2              | Image                        | /var/www/vendor/ezsystems/behatbundle/src/lib/Data/Images/small1.jpg |
```
Comment (mnocon) [inline features/examples/content.feature]: Still not sure if the format here should be <fullPathToFile> or <relativeToMetarepoPath> - the second one is harder to use for files outside metarepo, but more universal (full path won't pass on local environment most likely)

File: src/lib/API/ContentData/FieldTypeData/EmailDataProvider.php [hunk 2]
```diff
@@ -19,4 +19,9 @@ public function generateData(string $language = 'eng-GB'): string
 
         return $this->getFaker()->companyEmail;
     }
+
+    public function parseFromString(string $value)
+    {
+        return $value;
+    }
```
Comment (m-tyrala) [inline src/lib/API/ContentData/FieldTypeData/EmailDataProvider.php]: Maybe we can change base interface into base class and avoid such repetitions?

Comment (mnocon) [reply src/lib/API/ContentData/FieldTypeData/EmailDataProvider.php]: Refactored in https://github.com/ezsystems/BehatBundle/pull/89/commits/3cc58681055009fedb9bfe25ed02fbbeca1ec73e, introduced AbstractFieldTypeDataProvider that has the default implementation and gives access to Faker

**Comment by mnocon** [comment]
@lserwatka this one can be merged

**Pull request merged**